unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WalletHUD/Tests/WalletCardHUDComponentViewShould.cs

```diff
@@ -0,0 +1,130 @@
+﻿using NUnit.Framework;
+using System.Globalization;
+using UnityEditor;
+using UnityEngine;
+
+namespace DCL.Wallet
+{
+    public class WalletCardHUDComponentViewShould
+    {
+        private WalletCardHUDComponentView walletCardHUDView;
+
+        [SetUp]
+        public void SetUp()
+        {
+            WalletCardHUDComponentView prefab = AssetDatabase.LoadAssetAtPath<WalletCardHUDComponentView>(
+                "Assets/Scripts/MainScripts/DCL/Controllers/HUD/WalletHUD/Prefabs/WalletCard.prefab");
+
+            walletCardHUDView = Object.Instantiate(prefab);
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            walletCardHUDView.Dispose();
+        }
+
+        [Test]
+        public void RaiseOnWalletCardClickedFromGuestButtonCorrectly()
+        {
+            // Arrange
+            var walletCardClicked = false;
+            walletCardHUDView.OnWalletCardClicked += () => walletCardClicked = true;
+
+            // Act
+            walletCardHUDView.guestButton.onClick.Invoke();
+
+            // Assert
+            Assert.IsTrue(walletCardClicked);
+        }
+
+        [Test]
+        public void RaiseOnWalletCardClickedFromSignedInWalletButtonCorrectly()
+        {
+            // Arrange
+            var walletCardClicked = false;
+            walletCardHUDView.OnWalletCardClicked += () => walletCardClicked = true;
+
+            // Act
+            walletCardHUDView.signedInWalletButton.onClick.Invoke();
+
+            // Assert
+            Assert.IsTrue(walletCardClicked);
+        }
+
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public void SetWalletCardActiveCorrectly(bool isActive)
+        {
+            // Act
+            walletCardHUDView.SetWalletCardActive(isActive);
+
+            // Assert
+            Assert.AreEqual(isActive, walletCardHUDView.gameObject.activeSelf);
+        }
+
+        [Test]
```

Comment:
you don't need` [Test]` attribute when you have` [TestCase]`

Reply:
I didn't know! Thanks for the tip.

Done here: https://github.com/decentraland/unity-renderer/pull/5318/commits/b3003f328c165a6ad3a11b04256bd1e48c1cc0d0